k8s/demo/crunchy-postgres/README.md

```diff
@@ -1,35 +1,203 @@
-This is place-holder for providing the steps to run clustered postgresql on K8s+OpenEBS. We hope to merge these files back to https://github.com/CrunchyData/crunchy-containers/blob/master/examples/kube/
+# Running Crunchy-Postgres with OpenEBS
 
+This tutorial provides detailed instructions to run a PostgreSQL Statefulset with OpenEBS storage and perform 
+simple database operations to verify successful deployment. 
 
+## Crunchy-Postgres 
 
-## Usage
+The postgres container used in the Statefulset is sourced from (CrunchyData)[https://github.com/CrunchyData/crunchy-containers]. CrunchyData provides cloud agnostic PostgreSQL container technology that is designed for production 
+workloads with cloud native HA, DR, and monitoring.  
 
-Once you have OpenEBS enabled on your K8s Cluster, you can use the following simple steps to launch a clustered postgresql service with one master and one replica. 
+## Prerequisite
 
-### Step 1: Download the files to your host, which has access to kubectl 
+A fully configured (preferably, multi-node) Kubernetes cluster configured with the OpenEBS operator and OpenEBS
+storage classes.
+
+```
+test@Master:~/crunchy-postgres$ kubectl get pods
+NAME                                                             READY     STATUS    RESTARTS   AGE
+maya-apiserver-2245240594-ktfs2                                  1/1       Running   0          3h
+openebs-provisioner-4230626287-t8pn9                             1/1       Running   0          3h
+```
+
+## Deploy the Crunchy-Postgres Statefulset with OpenEBS storage
+
+The Statefulset specification JSONs are available at OpenEBS/k8s/demo/crunchy-postgres. 
+
+The number of replicas in the Statefulset can be modified in the *set.json* file. This example uses 2 replicas, 
+which includes one master and one slave. The Postgres pods are configured as primary/master or as replica/slave by 
+a startup script which decides the role based on ordinality assigned to the pod.
```

Comment:
ordinality ? Need to clarify with you. Will discuss.

Reply:
Ordinality is a property/value assigned by kubernetes to the constituent instances/pod of a statefulset. For example, in the pods below, 0 & 1 are the ordinality values. Based on the ordinality provided, the DB startup scripts configure it as primary/master OR as a slave/replica.

pgset-0
pgset-1